appyters/Enrichment_Analysis_Visualizer/Enrichment-Appyter.ipynb

```diff
@@ -372,20 +372,30 @@
     "    byte_string = bytearray(s, \"utf8\")\n",
     "    return int(hashlib.sha256(byte_string).hexdigest(), base=16) % i\n",
     "\n",
+    "def lighten(cols, step):\n",
+    "    '''\n",
+    "    Lightens a list of hex colors {cols} by {step} number of times. \n",
+    "    '''\n",
+    "    i = step\n",
+    "    while i != 0:\n",
+    "        cols = [int(col[1:], 16) for col in cols]\n",
+    "        cols = [hex((col & 0x7f7f7f) << 1).replace('0x', '#') for col in cols]\n",
```

Comment:
Please use `f"{x:06x}"` as the current approach does not seem guaranteed to produce strings that look like: #123456  -- they may also come out like #1234 (non-valid colors)